utils/lib/Alert.ts

```diff
@@ -297,6 +297,16 @@ class Alert extends Component<QuickstartConfigAlert[], QuickstartAlertInput[]> {
 
     return result;
   }
+
+  static getAll() {
+    const alertPaths = glob.sync(path.join(__dirname, '..', '..', 'alert-policies', '**', '*.+(yml|yaml)'));
+    return alertPaths.map(alertPath => {
+      // The identifier for alerts is the folder and the file name
+      // e.g. `node-js/HighCpuUtilization.yml`
+      const identifier = path.join(...alertPath.split('/').slice(-2, -1));
```

Comment:
I know it's not _likely_, but I always get a little anxious seeing `slice` (or `pop`) on arrays since those can technically return `[]` (and `undefined` for `pop`).

Typescript doesn't seem to be angry at this here, but I know that we're using a `!` in the Quickstart file to get around this. We might want to consider some additional error checking (or tests) to ensure we _for sure_ have an identifier.

Reply:
Note: I wouldn't block the PR on this!